guides/common/modules/ref_capsule-storage-requirements.adoc

```diff
@@ -15,8 +15,8 @@ endif::[]
 |Directory |Installation Size |Runtime Size
 ifdef::katello,satellite,orcharhino[]
 |/var/lib/pulp |1 MB |300 GB
+|{postgresql-lib-dir} |100 MB |10 GB
```

Comment:
This seems to be an actual change. Does this not affect Foreman and Foreman on Debian?

Reply:
On a Smart Proxy there is nothing that pulls in PostgreSQL. It's Foreman and Pulp that do.